Docker/gemini/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM golang
+
+ENV GEMINI_URL="https://github.com/scylladb/gemini/releases/download/v1.7.3/gemini_1.7.3_Linux_x86_64.tar.gz"
+
+RUN wget -O - $GEMINI_URL | tar -zxvf -
+
+CMD /gemini
```

Comment:
Am I nuts or does this not match up to what's in gemini-job.yaml?  It looks like in that case the binary is installed as /go/gemini but here it appears to be at /gemini... ?

Reply:
No, you're not nuts, good catch actually, the right path should be `/go/gemini` which is the default workdir for the golang image. Originally I had a differente base image.

Fixed!